css/products.css

```diff
@@ -0,0 +1,104 @@
+@import url("./index.css");
+
+.filters {
+  position: fixed;
+  width: 18rem;
+  height: 100vh;
+}
+
+.btn:hover {
+  transform: none;
+}
+
+.btn-sec {
+  outline: 2px solid var(--secondary-color);
+}
+
+.product-cards,
+.footer {
+  margin-left: 18rem;
+  width: calc(100% - 18rem);
+}
+
+.checkbox,
+.radiobutton {
+  width: fit-content;
+  transform: scale(1.2);
+}
+
+.filter-section {
+  height: 70vh;
+  overflow-y: scroll;
+}
+
+.card {
+  width: 18rem;
+}
+
+.card-img {
+  height: 150px;
+}
+
+.card-img img {
+  object-fit: scale-down;
+}
+
+.card {
+  height: 25rem;
+}
+
+.rating {
+  background-color: var(--primary-extra-light-color);
+  padding: var(--size-3) var(--size-5);
+  border-radius: var(--radius-large);
+}
+
+.strike-color {
+  color: var(--btn-error-color);
+}
+
+.grid-cols-auto {
+  grid-template-columns: repeat(auto-fit, minmax(18rem, 18rem));
```

Comment:
You can probably skip `minmax` here since both `min` and `max` values are the same.

Reply:
Valid point, current implementation seems redundant
Thank you for pointing out,have updated code